string robot_name
geometry_msgs/PoseStamped robot_goal
